### PR TITLE
MWPW-1224169 Updated How-to block

### PR DESCRIPTION
**Pull request by auniverseaway**
* Make HowTo more resilient to different content types
* Create layout that is more amenable to consumer styling

Resolves: [MWPW-124169](https://jira.corp.adobe.com/browse/MWPW-124169)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/cpeyer/howto?martech=off
- After: https://c3-howto-124169--milo--adobecom.hlx.page/drafts/cpeyer/howto?martech=off

### CODE REVIEW

File: libs/blocks/how-to/how-to.js [hunk 1]
```diff
@@ -50,20 +29,98 @@ function loadJsonLd(el) {
...
+  el.children[1].remove();
+  return steps;
+};
 
 export default function init(el) {
-  loadJsonLd(el);
+  const isSeo = el.classList.contains('seo');
+  const isLargeImage = el.classList.contains('large-image');
+
+  const { desc, heading, mainImage } = getHowToInfo(el);
```
Comment (rgclayton) [inline libs/blocks/how-to/how-to.js:108]: Nice!

File: libs/blocks/how-to/how-to.js [hunk 2]
```diff
@@ -50,20 +29,98 @@ function loadJsonLd(el) {
...
+  const { steps, images } = getHowToSteps(el);
+
```
Comment (rgclayton) [inline libs/blocks/how-to/how-to.js:110]: Great update.

**Review by rgclayton — APPROVED** [review]
Can confirm I am not seeing multiple json+ld scripts in the page any more. 

Image addition looks good.

**Comment by fullcolorcoder** [comment]
I think it looks good, but are we ditching the original formatting? I'm seeing a block not loaded in that case on https://c3-howto-124169--milo--adobecom.hlx.page/drafts/cpeyer/howto?martech=off

image

**Comment by auniverseaway** [comment]
@fullcolorcoder I just checked stock, and they are not using this feature.

I do think it will break some stuff, but it would only be (to my knowledge) non-essential demo Milo content.

**Pull request merged**